### ISSUE TITLE
'Drag-and-Drop' operation does not work in IE11

### ISSUE THREAD

**Issue by VoimiX** [autoNumeric/autoNumeric#518]
### Expected behavior
After dragging number to autonumeric field  it initializes correctly

### Actual behavior
After dragging number to autonumeric field an error occurs
```
SCRIPT87: Invalid argument. 
 var droppedText = e.dataTransfer.getData('text/plain');
```

### Steps to reproduce the problem
1. Using autoNumeric `4.0.3` and the browser `IE` version `11` on `Windows`,
2. I used the options
```js
     decimalCharacterAlternative: ',',    
     digitGroupSeparator: ' ',
     minimumValue: 0,
     modifyValueOnWheel: false,
```
3. Initialized my AutoNumeric object with the call
```js
 $("[data-money]").each(function(ndx, lmnt) {
   var n = new AutoNumeric(lmnt, { 
   	 decimalCharacterAlternative: ',',    
     digitGroupSeparator: ' ',
     minimumValue: 0,
     modifyValueOnWheel: false,
   });
 });
```
4. Drag any number or string to autonumeric textbox
5. An error occurs in a console
```
SCRIPT87: Invalid argument. 
 var droppedText = e.dataTransfer.getData('text/plain');
```

### Link to live example
https://jsfiddle.net/qm01xh6v/30/

**Comment by AlexandreBonneau** [comment]
This is due to...well...IE being IE (read: a ~shitty~obsolete browser) :/

A potential solution is evoked [here](https://stackoverflow.com/questions/18065840/html5-drag-and-drop-not-working-on-ie11).
The culprit being that while all browser understand the `'text/plain'` data type, IE only understand `'Text'`.

**Comment by VoimiX** [comment]
If IE doesnt understand this type may be just check it in the code and do nothing instead of exception?

**Comment by AlexandreBonneau** [comment]
My thinking is to check which browser is running the code, and modify the type from here.

**Comment by VoimiX** [comment]
ok)

**Comment by AlexandreBonneau** [comment]
Fixed in `v4.1.0-beta.13`
